### PR TITLE
Chat UI polish

### PR DESCRIPTION
**Pull request by toolmantim**
Various cleanups to the new chat UI implementation:

- Restyles the toolbar to better match VS Code
- Adds tooltips
- Fixes the sticky nav not staying stuck
- Switches back to vanilla shadcn/ui Collapsible by adding our own UI (composition over customisation of the vanilla shadcn/ui where possible)
- Animates collapsible transition
- Fixes button hover states and sizes in collapsible panel rows
- Fixes Account and Settings change changing/sticking when clicked

## Test plan

- Check storybooks
- Click around tabs in VS Code, trying different themes
- CI

### CODE REVIEW

File: vscode/webviews/components/ExpandableContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                    <Button variant="ghost" size="icon">
+                        <ChevronsUpDownIcon
+                            className="tw-h-8 tw-w-8 tw-text"
+                            size={16}
+                            strokeWidth="1.25"
+                        />
+                    </Button>
+                </CollapsibleTrigger>
+            </div>
+            <CollapsibleContent className="tw-overflow-hidden tw-transition-all data-[state=closed]:tw-animate-collapsible-up data-[state=open]:tw-animate-collapsible-down">
```
Comment (toolmantim) [inline vscode/webviews/components/ExpandableContainer.tsx:40]: We can probably push this up into our generic shadcn/ui component — can't imagine a time where you don't want it animated, given accordion is.

**Comment by toolmantim** [comment]
@vovakulikov please feel free to pop on any commits and merge at will, given the impending VS Code release! I won't be around for another 12 hours.

**Review by vovakulikov — APPROVED** [review]
Cody looks good to me, Checked this manually in VSCode build, didn't find any obvious problems with UI. Thanks

**Comment by vovakulikov** [comment]
I'm going to merge it to make sure this will be part of upcoming VSCode release

**Comment by toolmantim** [comment]
Thank you @vovakulikov!

**Pull request merged**